src/main/kotlin/org/radarbase/redcap/integration/RedCapIntegator.kt

```diff
@@ -80,11 +80,8 @@ class RedCapIntegator(private val redCapClient: RedCapClient) {
         )
     }
 
-    fun pullFieldsFromRedcap(fields: List<String>, recordId: Int) =
-        redCapClient.fetchFormDataForId(fields, recordId)
-            .filter { parser.canBeParsed(it.value) }
-            .mapValues { parser.parseField(it.value) }
-            .toMutableMap()
+    fun pullFieldsFromRedcap(fields: List<String>, recordId: Int, event: String? = null) =
+        redCapClient.fetchFormDataForId(fields, recordId, event)
 
     companion object {
         val parser = AttributeFieldParser()
```

Comment:
is this used anymore?

Reply:
Yes this is not used anymore, updated now.